CHANGES.txt

```diff
@@ -68,6 +68,7 @@
  * Add new CQL function maxWritetime (CASSANDRA-17425)
  * Add guardrail for ALTER TABLE ADD / DROP / REMOVE column operations (CASSANDRA-17495)
  * Rename DisableFlag class to EnableFlag on guardrails (CASSANDRA-17544)
+ * Nodetool bootstrap resume will now return an error if the operation fails (CASSANDRA-16491)
```

Comment:
Committer can fix this up on commit too, but make sure CHANGES bits go a the top of the appropriate section, not the bottom. (For a hint, look at the date progression of the `git blame` annotation in your IDE, etc.)